tools/c7n_azure/c7n_azure/resources/sqlauditingsettings.py

```diff
@@ -0,0 +1,56 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.from c7n_azure.provider import resources
+
+import logging
+
+from c7n_azure.provider import resources
+from c7n_azure.resources.arm import ChildArmResourceManager
+
+log = logging.getLogger('custodian.azure.sqlauditingsettings')
+
+
+@resources.register('sql-auditing-settings', aliases=['sqlauditingsettings'])
```

Comment:
this feels more appropriate as a filter then a child resource.

Reply:
not sure why this isn't showing up as a comment on the pr list view, adding another comment to trigger an update.